modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -81,31 +69,27 @@ Hover your mouse over a given point of a graph, to display an overlay of numeric
 image::monitoring-dashboards-hover.png[]
 
 
-If your server runs multiple applications, you can single out one or several and view the performance of these in a separate set of charts. To do so, select one or many, and a side-panel will display on the right with this data.
-
-image::monitoring-dashboards-39935.png[]
+If your server runs multiple applications, you can select applications in the list to view the performance of these in a separate set of charts. To do so, select the application and a side-panel will display on the right with this data.
 
-=== Metrics in the Server Dashboard
-== Server Metrics
+=== Server Metrics
 
-The graphs in this dashboard cover different metrics related to memory and CPU usage. You can see these two at a high level on the *Overview* tab, or you can dig into the *Breakdown* to see a detailed series of metrics that tracks each part of the memory separately.
+The graphs in this server's dashboard cover different metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab, or you can view the *Breakdown* to see a detailed series of metrics that tracks each part of the memory separately.
 
-At a high level, memory can be thought of as divided into what’s in the JVM heap and what’s outside of it.
+The graphs in this dashboard display memory usage levels in both the *Overview* and the *Breakdown* tabs, and track these values:
 
-The heap is the area where all Java objects reside. The heap is created when the JVM is started up, it can be scaled up or down in size without interrupting the applications in it. When the heap becomes full, garbage is collected. During the garbage collection, objects that are no longer used are cleared, thus making space for new objects.
-
-The non-heap memory stores per-class structures such as a runtime constant pool, field and method data, and the code for methods and constructors.
+* *Used* - The amount of memory (in MB) that is currently in use.
+* *Committed* - The amount of memory (in MB) that's available for use by the Java VM. This amount varies over time. The amount of committed memory is always greater than or equal to the amount of used memory.
+* *Total* - The maximum amount of memory (in MB) for use by the server. The maximum amount can change or be undefined. A memory allocation can fail if the Java VM attempts to increase the used memory to be greater than committed memory, even if the amount used is below the total.
 
-The graphs in this dashboard that display memory usage levels, in both the *Overview* and the *Breakdown* tabs, track the following three values:
+At a high level, you can think of memory as divided into what’s in the JVM heap and what’s outside of it.
```

Comment:
This sentence leaves me confused (I'm not sure if JVM heap is really different from what's measured in Committed or not. Nor am I sure why this is helpful. Or maybe move this to after you've explained what heaps are?

Reply:
Maybe lines 80-84 should be from the customer's point of view: what can the customer manipulate, what do they see that indicates they are heading toward a failure state. Also, I'm pretty sure memory allocation is guaranteed to fail (you don't need "can") if the JVM tries to increased used memory above committed? I thought that was why these things existed, to prevent that.